script/inputs/bsc-ynCoBTCk.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
separate pull request for this after the decimals one was merged

Reply:
Do you want this removed from this PR and added to a separate PR @danoctavian

Reply:
yes please that's cleaner